### DISCUSSION TITLE
Support another date format

### DISCUSSION THREAD

**Discussion by LorenzBischof** [srid/neuron#460] [Ideas]
Would it be possible to also support the following date format?:
```
date: 2020-11-11 12:24
```
When I run `neuron rib` I get the following error:
```
E 2011111224-note-title.html
  - 2011111224-note-title.md:1:0: error
       |
     1 | title: Note Title
       | ^
    parseTimeM: no parse of "2020-11-11 12:24"
```

**Comment by srid** [comment]
@lbischof Why not use `2020-11-11T12:43` which works (also what `neuron new` generates automatically)?

**Comment by LorenzBischof** [comment]
I am trying different tools and [vim-zettel uses that format by default.](https://github.com/michal-h21/vim-zettel/blob/master/autoload/zettel/vimwiki.vim#L170) I just prefer the format without the `T`, but I can change the default in vim-zettel.

**Comment by srid** [comment]
What do other Zettelkasten software use ? Zettlr, Obsidian, etc.
